src/IfSharp.Kernel/helpers/paket.dependencies

```diff
@@ -0,0 +1,4 @@
+source https://www.nuget.org/api/v2
+
+nuget FSharp.Data.TypeProviders 
+nuget XPlot.Plotly 
```

Comment:
It's almost like you could have all of FsLab (http://fslab.org) as a dependency.  Not quite sure of the ramifications of that though!